src/cli/main.go

```diff
@@ -18,6 +18,23 @@ type (
 	}
 )
 
+type meta_command_result bool
+
+const (
+	meta_command_success              meta_command_result = true
+	meta_command_unrecognized_command meta_command_result = false
+)
+
+func is_meta_command(inputbuffer *InputBuffer) meta_command_result {
+	if strings.Compare(inputbuffer.buffer, ".exit") == 0 {
+		fmt.Fprintf(os.Stdout, "Exitting Bye!")
+		os.Exit(0)
+	}
+
+	return meta_command_unrecognized_command
+
+}
+
```

Comment:
I think it's time to split functions to separate files.

Reply:
Ok In the next Commit I will do so.